docs/source/tutorials/translate-app.rst

```diff
@@ -99,13 +99,71 @@ and call it in your components for example in a try/Except statement :
 
 
 Update the translated dictionaries
------------------------------------
+----------------------------------
+
+.. note::
+
+    If a key is missing in the target language dictionary, the :code:`Translator` object (:code:`ms`) will automatically fallback to the "en" key in order to avoid error or non displayed messages
+
+Automatic
+^^^^^^^^^
+
+If your application is part of the OpenForis initiative and hosted on SEPAL, you can request to add your project to the **Pontoon** application list. Pontoon is an open-source translation solution that will deal with the trouble of creating the files and keep the keys updated. To learn more, please see their `documentation <https://mozilla-l10n.github.io/localizer-documentation/tools/pontoon/>`__. From the developer's side you'll need to add the folder corresponding to the language you want to support and open a request for a new project in our `issue tracker <https://github.com/12rambau/sepal_ui/issues/new/choose>`__.
```

Comment:
If I understood it well... we need to open the issue on the sepal_ui tracker, so you can add a new project in pontoon, but how can we wire our repo with the project?

Reply:
The only person that will be requesting access will be people that are not us and that have read the documentation. So what I would say is "thank you for reading, open an issue and I'll let you know" (like adding a new lib on conda-forge).
I think I'll need to create a `sepal-bot` profile. So that people can add it as a maintainer, but I don't know the adress yet.